### DISCUSSION TITLE
@Play and Nacon Xbox controllers on Switch

### DISCUSSION THREAD

**Discussion by TC1995** [cathery/sys-con#136] [Controller requests]
**Description of the controller**
These controllers support USB but not Bluetooth, originally for PC usage.
When I insert these controllers on my Nintendo Switch, they are recognized, but somehow don't receive any input when pressing buttons or the D-Pad for example. 
What do I do to fix this issue?

Nacon:
![9200000079159857](https://user-images.githubusercontent.com/9269715/73183293-dc590c80-411a-11ea-9409-cf5e43f87bdd.jpg)

(at)Play
![Xbox-One-Translucent-Black-Wired-Controller](https://user-images.githubusercontent.com/9269715/73183294-dc590c80-411a-11ea-8335-6074c7e734c5.png)

**Comment by naxiv93** [comment]
I've got a Nacom pad. I ran the diagnostics tool, heres the screenshot.
Quite interested into seeing what's the peculiarity with this one. :)

![Switch diag](https://user-images.githubusercontent.com/38695848/78830467-4b3ec580-79e8-11ea-8b70-0d384ff5a63d.jpg)
